ncl/ncl_entries/trigonometric_functions.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>Does this need an update?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Nevermind, I see what's going on now.